e2e/test/MethodE2ETests.cs

```diff
@@ -126,6 +127,31 @@ public async Task Method_ServiceSendsMethodThroughProxyWithCustomTimeout()
             await SendMethodAndRespondAsync(Client.TransportType.Mqtt_Tcp_Only, SetDeviceReceiveMethodAsync, TimeSpan.FromMinutes(5), serviceClientTransportSettings).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task Method_ServiceInvokeDeviceMethodWithUnknownDeviceThrows()
+        {
+            // setup
+            using var serviceClient = ServiceClient.CreateFromConnectionString(Configuration.IoTHub.ConnectionString);
+            var methodInvocation = new CloudToDeviceMethod("SetTelemetryInterval");
+            methodInvocation.SetPayloadJson("10");
+
+            // act
+            ErrorCode actualErrorCode = ErrorCode.InvalidErrorCode;
```

Comment:
Q - does this need to be initialized?

Reply:
looks like an Enum, no initialization required

Reply:
not removed?